src/query/functions/src/scalars/timestamp/src/datetime.rs

```diff
@@ -1524,6 +1524,12 @@ fn register_to_number_functions(registry: &mut FunctionRegistry) {
         }),
     );
 
+    registry.register_passthrough_nullable_1_arg::<TimestampType, Float64Type, _, _>(
+        "epoch_microsecond",
```

Comment:
val is already in `microsecond`, the domain should be same as input.

Reply:
Done